centreon/tests/php/App/Upgrade/Infrastructure/Double/FakeUpdateScriptFinder.php

```
@@ -21,11 +21,17 @@
 
-use Core\Application\Common\UseCase\AbstractPresenter;
-use Core\Platform\Application\UseCase\UpdateVersions\UpdateVersionsPresenterInterface;
+use App\Upgrade\Domain\Repository\UpdateScriptFinder;
 
-class UpdateVersionsPresenter extends AbstractPresenter implements UpdateVersionsPresenterInterface
+final class FakeUpdateScriptFinder implements UpdateScriptFinder
 {
+    /** @var string[] */
+    public array $availableUpdates = [];
+
+    public function findOrderedAvailableUpdates(string $currentVersion): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$currentVersion'. (undefined)

(UnusedFormalParameter)